extensions/amp-subscriptions/0.1/local-subscription-platform-renderer.js

```diff
@@ -57,11 +57,22 @@ export class LocalSubscriptionPlatformRenderer {
     ]);
   }
 
+  /**
+   * Resets all the rendered content back to normal.
+   * @return {!Promise}
+   */
+  reset() {
+    // Close dialog. Ignored if the dialog is not currently open.
+    this.dialog_.close();
+    // Hide subscriptions sections.
+    return this.renderActionsInNode_({}, this.rootNode_, () => false);
+  }
+
   /**
    * @param {!JsonObject} renderState
    */
   renderActions_(renderState) {
-    this.renderActionsInNode_(renderState, this.rootNode_);
+    this.renderActionsInNode_(renderState, this.rootNode_, evaluateExpr);
```

Comment:
Where does `evaluateExpr` come from?

Reply:
It's imported above - a function in another module.

Reply:
ah ok